### DISCUSSION TITLE
tutorials/lift-off-part5/06-field-deprecation

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#159] [course feedback]
# tutorials/lift-off-part5/06-field-deprecation

https://www.apollographql.com/tutorials/lift-off-part5/06-field-deprecation

**Comment by normanmurillo-abbvie** [comment]
Is it possible to deprecate an entire query or mutation of behalf of another?

**Reply by lizhennessy** [reply]
Hi there @normanmurillo-abbvie ! The `@deprecated` directive is intended only for [fields and enum values](http://spec.graphql.org/June2018/#sec-Deprecation).
